composer/utils/misc.py

```diff
@@ -100,3 +112,37 @@ def using_torch_2_0_1() -> bool:
         bool: Return True if current version is greater than or equal to 2.0.1 else False
     """
     return version.parse(torch.__version__) >= version.parse('2.0.1')
+
+
+def validate_save_planner(save_planner: Optional[Any]) -> None:
```

Comment:
should these functions validate that we're on a valid torch version before attempting the import? they're defined as public, so should probably do that validation even if we only call them on paths where we have already checked torch version.